docs/install-gpu-pip.md

```diff
@@ -0,0 +1,10 @@
+```console
+pip install -U pip
+pip install -U setuptools wheel
+
+# Install the proper version of PyTorch following https://pytorch.org/get-started/locally/
+pip install torch==1.13.1+cu116 torchvision==0.14.1+cu116 --extra-index-url https://download.pytorch.org/whl/cu116
```

Comment:
I thought we didn't need to manually install torch for GPU? Shouldn't it install automatically via `pip install autogluon`?

Reply:
My intention was for these to be the exact same as they are on the existing website. I am not sure if your statement about not needing to manually install torch for GPU is correct but can look into it later.